<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>9</y>
      <width>250</width>
      <height>400</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>400</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>400</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
       <widget class="QSlider" name="sld_col">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>0</y>
          <width>31</width>
          <height>251</height>
         </rect>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>5</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="sld_row">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>108</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>5</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="row_value">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>100</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="col_value">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>230</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="statusLine">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="stateText"/>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>50</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_Shift">
          <property name="text">
           <string>Shift</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Action">
          <property name="text">
           <string>Action</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_resetWindow">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>227</width>
      <height>400</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>400</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="serverContent"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineString"/>
       </item>
       <item>
        <widget class="QPushButton" name="send">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="KeyCodeState">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>471</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
